index.js

```diff
@@ -66,7 +61,18 @@ module.exports = (options = {}) => {
     },
 
     load(plugin) {
+      const robot = createRobot({app, cache, logger, catchErrors: true});
+
+      // Connect the router from the robot to the server
+      server.use(robot.router);
```

Comment:
@bkeepers I think this is a really nice solution. 👍  We could have `robot.route` throw a misconfiguration error or something if nothing is passed in `roboto.route()` -> `throw new Error('Please provide a route prefix {link to docs}');` If you really do intend to bind with no prefix you could pass in `robot.route('')` or we could make it really explicit by using a constant on the robot class like `robot.route(robot.ROUTE_NO_PREFIX)`

I was thinking about how to get around not really knowing the plugin name but I think this is more elegant than anything I could come up with.